Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+class TopTabsViewController: UIViewController {
```

Comment:
Using a `UIViewController` here is smart 👍 Will help keep the code for the top tabs out of the ever-increasing mess of the BVC.